<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDashboardview</class>
 <widget class="QMainWindow" name="AdminDashboardview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="usersLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Users</string>
    </property>
   </widget>
   <widget class="QLabel" name="orderdLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Orders</string>
    </property>
   </widget>
   <widget class="QLabel" name="revenueLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Revenue</string>
    </property>
   </widget>
   <widget class="QWidget" name="salesChartWidget" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>521</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addNewProduct">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
